libraries/Servo/ServoTimers.h

```diff
@@ -0,0 +1,122 @@
+/*
+  Copyright (c) 2015 Arduino LLC. All right reserved.
+
+  This library is free software; you can redistribute it and/or
+  modify it under the terms of the GNU Lesser General Public
+  License as published by the Free Software Foundation; either
+  version 2.1 of the License, or (at your option) any later version.
+
+  This library is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  Lesser General Public License for more details.
+
+  You should have received a copy of the GNU Lesser General Public
+  License along with this library; if not, write to the Free Software
+  Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA
+*/
+
+/*
+ * Defines for 16 bit timers used with  Servo library
+ *
+ * If _useTimerX is defined then TimerX is a 16 bit timer on the current board
+ * timer16_Sequence_t enumerates the sequence that the timers should be allocated
+ * _Nbr_16timers indicates how many 16 bit timers are available.
+ */
+
+#ifndef __SERVO_TIMERS_H__
+#define __SERVO_TIMERS_H__
+
+#include "nrf.h"
+
+/**
+ * NRF5 Only definitions
+ * ---------------------
+ */
+
+// Uncomment to select allocatable timers, 3 and 4 are only available in nRF52
+//#define _useTimer0
```

Comment:
Are you sure it's just the RTC0? I haven't read the soft device documentation, but looking at this it sounds like it might also use TIMER0:
[cores/nRF5/SDK/components/softdevice/s132/headers/nrf_nvic.h#L76-L110](https://github.com/sandeepmistry/arduino-nRF5/blob/8747fa2d5947dd3cde1d94abf796dd2c17781177/cores/nRF5/SDK/components/softdevice/s132/headers/nrf_nvic.h#L76-L110)